src/pages/InstitutionSettings/InstitutionSettings.tsx

```diff
@@ -20,6 +22,7 @@ const InstitutionSettings: FC = () => {
   const { institution } = useInstitutionFetch({
     id: institutionId,
   })
+  const { institutionVacations } = useInstitutionVacationsFetch()
```

Comment:
**[non-blocker]**

This could also be inside InstitutionForm. Seems that institutionVacations is only needed inside InstitutionForm